day_7/fizzbuzz.js

```diff
@@ -0,0 +1,25 @@
+// First part:
+ for (var i = 1; i <= 100; i++)
+  if (i % 15 == 0)
```

Comment:
Interesting approach here - I've usually seen line 3 written as `if (i % 3 === 0 && i % 5 === 0)`. Keep up the good work!